changelog.md

```diff
@@ -61,26 +97,54 @@ We now publish a separate marathon-plugin-interface.jar with every Marathon rele
 This artifact holds all the inerfaces needed to develop your own Marathon plugin.
 
 #### Consolidate logging to use slf4j
-We moved completely to slf4j as Logging API. The logging backend still uses log4j.
+We moved completely to slf4j as Logging API. 
+
+#### Several performance improvements
+* A separate thread pool is used for health check operations
+* Efficiently create groups/apps 
```

Comment:
Group and app creation is more efficient.
(just a stylistic tweak to make it match the context)